src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,20 @@
+export const Footer = () => {
+  return (
+    <footer class="footer-main">
+      <div class="copy-right">
+        <i class="fal fa-copyright"></i> 2022 UI Inc. All rights reserved.
+      </div>
+      <div class="icon-container">
+        <a class="footer-icon" href="">
+          <i class="fab fa-github"></i>
+        </a>
+        <a class="footer-icon" href="">
+          <i class="fab fa-twitter"></i>
+        </a>
+        <a class="footer-icon" href="">
+          <i class="fab fa-linkedin-in"></i>
```

Comment:
Provide links here in href and you can make a common component for this t avoid code repetition.

Reply:
done